helpers/template.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash'
+
+/**
+ * Creates list of comma separated values
+ * @param {Array<String>} arr
+ * @returns {String}
+ */
+function strArrToStrList (arr) {
+  return _.reduce(arr, (str, item) => `${str}, ${item}`)
+}
+
+/**
+ * Returns string with dots if
+ * length of a string is too big
+ * @param {String} str
+ * @param {Number} [length=3]
+ * @returns {String}
+ */
+function trimString (str, length = 3) {
+  return (length - 3) >= str.length ? str : `${str.slice(0, length - 3)}...`
```

Comment:
I prefer to rename this function to `ellipsis(str, length = 3)`, because this is not only trimming but making 'wielokropek' too